src/test/scala/nl/biopet/utils/ngs/fastq/ValidationTest.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2014 Biopet
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package nl.biopet.utils.ngs.fastq
+
+import htsjdk.samtools.fastq.FastqReader
+import nl.biopet.test.BiopetTest
+import org.testng.annotations.Test
+
+import scala.collection.JavaConversions._
+class ValidationTest extends BiopetTest {
+
+  @Test
+  def testValidPairs(): Unit = {
+    val r1 = new FastqReader(resourceFile("/R1.fq"))
```

Comment:
To isolate test it's maybe better to just generate the test fastq records here instead of reading a file? This because the method only accepts a single pair and not a file.